wordpress-mobile/gutenberg-mobile
# Reduce the number of items per page when fetching reusable blocks

`gutenberg` PR: https://github.com/WordPress/gutenberg/pull/29626

To test:
1. Add more than 27 reusable blocks to a site (28 items should trigger the exception), this has to be done in the web version because creating reusable blocks from the app is not yet available.
1. Open a post from the WPAndroid app.
1. Observe that no crash or exception is produced.

PR submission checklist:
